guidelines/index.html

```diff
@@ -3210,13 +3210,13 @@ <h2 id="x6-glossary"><span class="secno">6. </span>用語集<span class="permali
    <p><em>例外:</em> ホワイトノイズや、1 辺が (典型的な閲覧距離における視野の) 0.1 度未満の市松模様のような細かくバランスの取れた閃光は、閾値を超えることにはならない。
    </p>
    
-   <div class="note" id="issue-container-generatedID-83"><div role="heading" class="note-title marker" id="h-note-83" aria-level="3"><span>注記</span></div><p class="">一般的なソフトウェア又はウェブコンテンツの場合、コンテンツを 1024 × 768 ピクセルで表示したときに画面上の任意の場所で 341 × 256 ピクセルの矩形を使用すると、標準的な画面サイズ及び視聴距離 (例: 15～17 インチの画面を 22～26 インチの距離で視聴) における視野 10 度の適切な見積もりとなる (同じコンテンツでも高解像度のディスプレイでは小さく安全になるので、閾値を定めるには低解像度が用いられている)。
+   <div class="note" id="issue-container-generatedID-83"><div role="heading" class="note-title marker" id="h-note-83" aria-level="3"><span>注記</span></div><p class="">一般的なソフトウェア又はウェブコンテンツの場合、コンテンツを 1024 × 768 ピクセルで表示したときに画面上の任意の場所で 341 × 256 ピクセルの矩形を使用すると、標準的な画面サイズ及び視聴距離 (例: 15～17 インチの画面を 22～26 インチの距離で視聴) における視野 10 度の適切な見積もりとなる。この 75～85 ppi という解像度は、CSS 仕様の公称ピクセル解像度である 96 ppi よりも低く、したがってより保守的であると知られている。同じコンテンツのレンダリングを表示する高解像度のディスプレイは、より小さく安全な画像が得られるため、閾値値の定義に使用される解像度はより低くなる。
```

Comment:
とりあえずの感想:

- この nominal を「公称」とするのは少し違和感があります。「名目」? 「規定」?
- 「保守的であると知られている」というのも違和感が。原文だと "known to be lower" とくっついていて、「低いことが知られている」というのは違和感がないのですが。語順の問題のような気もしつつ、「知られている」はなくても良いような気もしつつ。
- 「閾値値」は「閾値」ですかね?
- 「高解像度のディスプレイは、より小さく安全な画像が得られる」は日本語としての係受けがおかしいかも。yieldをそのまま訳すなら「より小さく安全な画像をもたらす」という感じでしょうか。「高解像度のディスプレイでは」としても良いと思います。語順を多少入れ替えると読みやすいかもしれません。
- 最後の「閾値の定義に使用される解像度はより低くなる」が高解像度ディスプレイに限定された話に読めるので、すこし調整した方がいいかも。「だから閾値の定義には低めの解像度を使うようにしている」ということだと思うのですが。

といったところでこのようにしてみてはどうでしょうか。

```suggestion
   <div class="note" id="issue-container-generatedID-83"><div role="heading" class="note-title marker" id="h-note-83" aria-level="3"><span>注記</span></div><p class="">一般的なソフトウェア又はウェブコンテンツの場合、コンテンツを 1024 × 768 ピクセルで表示したときに画面上の任意の場所で 341 × 256 ピクセルの矩形を使用すると、標準的な画面サイズ及び視聴距離 (例: 15～17 インチの画面を 22～26 インチの距離で視聴) における視野 10 度の適切な見積もりとなる。この 75～85 ppi という解像度は、CSS 仕様の規定ピクセル解像度である 96 ppi よりも低いことが知られており、したがってより保守的である。高解像度のディスプレイでは、同じコンテンツのレンダリングを表示するとより小さく安全な画像が得られるため、閾値の定義には低めの解像度を使用している。
```

Reply:
@bakera ありがとうございます。改めて原文と併せてプルーフリーディングしてみましたが、いただいた修正提案にて、異論ありませんです。